<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="soccf-tr5-001" name="component-tests(failed)(failed)" tests="2" failures="1" timestamp="8 Oct 2014 10:50:42 GMT" time="10.579" errors="0">
  <testcase name="testAll" time="0.203" classname="teammates.test.cases.logic.EvaluationsLogicTest">
    <failure type="java.lang.AssertionError" message="expected:&amp;lt;1&amp;gt; but was:&amp;lt;0&amp;gt;">
      <![CDATA[java.lang.AssertionError: expected:<1> but was:<0>
	at teammates.test.cases.logic.EvaluationsLogicTest.testGetEvaluationsClosingWithinTimeLimit(EvaluationsLogicTest.java:273)
	at teammates.test.cases.logic.EvaluationsLogicTest.testAll(EvaluationsLogicTest.java:82)
	at java.util.concurrent.FutureTask.run(FutureTask.java:262)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)
... Removed 21 stack frames]]>
    </failure>
  </testcase> <!-- testAll -->
  <testcase name="testAll" time="6.1" classname="teammates.test.cases.logic.FeedbackSessionsLogicTest"/>
</testsuite> <!-- component-tests(failed)(failed) -->
